build.sbt

```diff
@@ -353,6 +354,23 @@ lazy val jdkStreamsBenchmarks = (project in file("benchmarks/jdk-streams"))
   )
   .dependsOn(renaissanceCore % "provided")
 
+lazy val http4kBenchmarks = (project in file("benchmarks/http4k"))
+  .enablePlugins(KotlinPlugin)
+  .settings(
+    name := "http4k",
+    commonSettingsNoScala,
+    kotlinVersion := "2.0.0",
+    kotlincJvmTarget := "21",
```

Comment:
I did not have any strong reasons to have it other than optimized memory management etc. If compatibility is the priority then I do not have any reason to leave it on 21. Changed.

Reply:
Thanks! So perhaps can we patch this at compile time somehow with the "current" JVM version? Because default seems to be 1.6 which is probably too conservative for us.

@lbulej What do you think about which version to put here when generating the bytecode?